style.css

```
@@ -0,0 +1,109 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #f4f4f4;
+    font-family: Arial, sans-serif;
+    color: #333;
+    margin: 0 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+header {
+    background-color: #547dd0;
+    display: flex;
+    align-items: center;
+    padding: 20px;
+    width: 100%;
+    max-width: 900px;
+    border-radius: 10px;
+}
+
+header img {
+    width: 250px;
+    height: 150px;
+    border-radius: 10%;
+    margin-right: 20px;
+}
```

```suggestion
header {
    background-color: #547dd0;
    display: flex;
    align-items: center;
    padding: 20px;
    width: 100%;
    max-width: 900px;
    border-radius: 10px;
    
        img {
            width: 250px;
            height: 150px;
            border-radius: 10%;
            margin-right: 20px;
        }
}

```

Se puede usar css nesting